_includes/cockroachcloud/userfile-examples/import-into-userfile.md

```diff
@@ -0,0 +1,33 @@
+To import from `userfile`, first create the table that you would like to import into:
+
+{% include copy-clipboard.html %}
+~~~sql
+CREATE TABLE customers (
+  id INT,
+  dob DATE,
+  first_name STRING,
+  last_name STRING,
+  joined DATE
+);
+~~~
+
+Then, use `IMPORT INTO` to import data into the table:
```

Comment:
IMPORT INTO does not work on CC. I tried it this morning. We should recommend "IMPORT TABLE..." for now.

https://cockroachlabs.slack.com/archives/CJXBR3693/p1626954455022800

Reply:
yes! I defo went into `IMPORT TABLE` deprecated mode after just finishing that PR, thanks for raising.